### PR TITLE
Register new package Hermetic.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [gragusa/Hermetic.jl](https://github.com/gragusa/Hermetic.jl)
Release: [v0.1.0](https://github.com/gragusa/Hermetic.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/gragusa/Hermetic.jl/builds/235937988)
cc: @gragusa

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@gragusa This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

File: Hermetic/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+Compat 0.25.2
```
Comment (tkelman) [inline Hermetic/versions/0.1.0/requires]: doesn't look like you're using anything newer than `@compat abstract type`, which was in 0.17

**Comment by tkelman** [comment]
would be good to fill in the package description to something other than "No description, website, or topics provided."

**Comment by gragusa** [comment]
@tkelman thanks! Updated REQUIRE and added brief description.

**Comment by tkelman** [comment]
If the code is based on a translation of existing code, should make that more explicit in the license file.

**Pull request merged**